tests/e2e/specs/empty-trash-restore-trashed-posts.test.js

```diff
@@ -0,0 +1,71 @@
+import {
+    visitAdminPage,
+    createNewPost,
+    trashAllPosts,
+    publishPost
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Empty Trash', () => {
+
+
+    async function createPost() {
+
+        // Create a Post
+        const title = 'Test Title';
+        await createNewPost( { title } );
+        await publishPost();
+
+    }
+
+
+    afterEach( async () => {
+        await trashAllPosts();
+    } );
+
+    it( 'Empty Trash', async () => {
+
+        await createPost();
+       
+        await visitAdminPage( '/edit.php' );  
+
+        // Move post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Empty trash 
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");  
+        const deleteAllButton = await page.waitForSelector('input[value="Empty Trash"]');
+        await deleteAllButton.click();
+        await page.waitForSelector("#message");
+        
+    } );
+
+
+    it( 'Restore trash post', async () => {
+
+        await createPost();
+    
+        await visitAdminPage( '/edit.php' );  
+
+        // Move one post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Remove post from trash 
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");
```

Comment:
Same here.

Reply:
This feedback is addressed, some we tried to use xpath as suggested by you but unfortunately using xpath selector is leading to failing CI, in my local it is working fine but in CI its failing.